zqd/api/api.go

```diff
@@ -112,9 +112,15 @@ type PacketPostStatus struct {
 
 type LogPostRequest struct {
 	Paths          []string             `json:"paths"`
+	StopErr        bool                 `json:"stop_err"`
 	JSONTypeConfig *ndjsonio.TypeConfig `json:"json_type_config"`
 }
 
+type LogPostWarning struct {
+	Type string `json:"type"`
+	Msg  string `json:"msg"`
```

Comment:
This would make it consistent with `SearchWarnings`:
```suggestion
	Warnings  []string `json:"warnings"`
```
though I'm not sure what the rationale is to have a slice of string instead of a single string.

Reply:
I noticed that, but also not sure of the rationale for slice in `SearchWarnings`. Currently I don't think there's a way for search to send multiple warnings in one message. Still I'll adjust as per your suggestion as it gives us the option to send multiple in the future.